tests/transformer/test_try.py

```diff
@@ -47,6 +50,34 @@ def test_RestrictingNodeTransformer__visit_Try__2(
     ])
 
 
+TRY_EXCEPT_STAR = """
+def try_except_star(m):
+    try:
+        m('try')
+        raise ExceptionGroup("group", [IndentationError('f1'), ValueError(65)])
+    except* IndentationError:
+        m('IndentetionError')
```

Comment:
```suggestion
        m('IndentationError')
```
I guess this is an accidental typo (there's the same on line 76)

Reply:
ok :)

Reply:
Fixed